lib/Dialect/TorchConversion/Transforms/Passes.cpp

```diff
@@ -96,7 +96,11 @@ void TorchConversion::createTorchBackendToLinalgOnTensorsBackendPipeline(
 }
 
 void TorchConversion::createTorchBackendToTosaBackendPipeline(
-    OpPassManager &pm) {
+    OpPassManager &pm, const TosaBackendPipelineOptions &options) {
+  if(!options.customOps.empty()){
```

Comment:
nit: run `git clang-format`. Spacing is off

Reply:
```
(mlir_venv) nod% git clang-format .
clang-format did not modify any files
```

Reply:
`git clang-format` will only modify staged files. To modify the files already committed, you need to specify the range of commits. For example: `git clang-format HEAD~1`

Reply:
Done